biosimulators_utils/omex_meta/utils.py

```diff
@@ -51,12 +51,16 @@ def build_omex_meta_file_for_model(model_filename,
         raise ValueError('Metadata could not be extracted from model `{}`:\n  {}'.format(
             model_filename, '\n'.join(errors).replace('\n', '\n  ')))
 
-    model = editor.get_xml()
+    model = None
+    try:
+        model = editor.get_xml()
+    except:
```

Comment:
There's lint error here
```
./biosimulators_utils/omex_meta/utils.py:57:5: E722 do not use bare 'except'
```

Reply:
We're trying to be more specific rather than just catching everything. Can the specific type of exception be inserted here?

```python
except <TypeOfException`> as exception:
```

Ideally the exception message would be incorporated in line 61 below unless its uniformative.

Reply:
Makes sense.  This particular error was something low-level; I think it was an invalid read on my machine  I'll figure out what it was and revise.  

However, looking at the githubactions logs, it's an actual Segmentation Fault on linux, so that might be more problematic.  Fortunately, it only happened with biomodels 585, and there because it wrote out a series of metadataN.xml files which later got confused for SBML files (I think in the fix-entries code); this error is actually the result of reading in a metadata file and trying to parse it as SBML.  Obviously something that needs to be fixed on that end, but fortunately something I can fix on our end for the time being.

Reply:
Segmentation faults are what I saw before. Previously I called pyomexmeta through a subprocess to capture this.

Its probably easiest to change the SBML than to wait for pyomexmeta.